internal/namespaces/registry/v1/custom_docker_helper.go

```diff
@@ -0,0 +1,247 @@
+			{
+				Name:  "helper-directory",
+				Short: "Directory in which the Docker helper will be installed",
+			},
+		},
+		Run: registryDockerHelperConfigureRun,
+	}
+}
+
+func registryDockerHelperConfigureRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+	// TODO add windows support
+	if runtime.GOOS == "windows" {
+		return nil, fmt.Errorf("windows is not currently supported")
+	}
+
+	binaryName := core.ExtractBinaryName(ctx)
+
+	_, _ = interactive.Println("To enable the Docker credential helper, scw needs to create a script inside of your $PATH.")
+	scriptDirArg := argsI.(*registryDockerHelperConfigureRequest).HelperDirectory
+	if scriptDirArg == "" {
+		defaultScriptDirArg := filepath.Join(filepath.Dir(scw.GetConfigPath()), "bin")
+
+		promptedScriptDir, err := interactive.PromptStringWithConfig(&interactive.PromptStringConfig{
+			Prompt:          "In which directory do you want to install the script?",
+			DefaultValue:    defaultScriptDirArg,
+			DefaultValueDoc: defaultScriptDirArg,
+		})
+		if err != nil {
+			return nil, err
+		}
+		scriptDirArg = promptedScriptDir
+	}
+	profileFlag := ""
+	profileName := core.ExtractProfileName(ctx)
+	if profileName != "" {
+		profileFlag = fmt.Sprintf(" --profile %s", profileName)
+	}
+
+	helperScriptPath := filepath.Join(scriptDirArg, fmt.Sprintf("docker-credential-%s", binaryName))
+	helperScript := fmt.Sprintf("#!/bin/sh\n\n%s%s registry docker-helper \"$@\"", binaryName, profileFlag)
+
+	// Warning
+	_, _ = interactive.Println()
+	_, _ = interactive.PrintlnWithoutIndent("To enable the Docker credential helper we need to create the file " + helperScriptPath + " with the following lines:")
+	_, _ = interactive.Println(helperScript)
+
+	// Early exit if user disagrees
+	_, _ = interactive.Println()
+	continueInstallation, err := interactive.PromptBoolWithConfig(&interactive.PromptBoolConfig{
+		Prompt:       fmt.Sprintf("Do you want to proceed with these changes?"),
+		DefaultValue: true,
+	})
+	if err != nil {
+		return nil, err
+	}
+	if !continueInstallation {
+		return nil, fmt.Errorf("installation cancelled")
+	}
+
+	err = writeHelperScript(helperScriptPath, helperScript)
+	if err != nil {
+		return nil, fmt.Errorf("failed to write helper script: %s", err)
+	}
+
+	var registries []string
+	for _, region := range scw.AllRegions {
+		registries = append(registries, endpointPrefix+region.String()+endpointSuffix)
+	}
+
+	err = setupDockerConfigFile(registries, binaryName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to write docker config file: %s", err)
+	}
+
+	_, _ = interactive.Println()
+	_, err = exec.LookPath(fmt.Sprintf("docker-credential-%s", binaryName))
+	if err != nil {
+		_, _ = interactive.Println(fmt.Sprintf("docker-credential-%s is not present in your $PATH, you should add %s to your $PATH to make it work.", binaryName, helperScriptPath))
+	} else {
+		_, _ = interactive.PrintlnWithoutIndent("Docker credential helper successfully installed.")
```

Comment:
Could you add more information about what to do from here? How can I test it works?